inc/sairouterintf.h

```diff
@@ -94,7 +100,7 @@ typedef enum _sai_router_interface_attr_t
      * @type sai_object_id_t
      * @objects SAI_OBJECT_TYPE_VLAN
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
-     * @condition SAI_ROUTER_INTERFACE_ATTR_TYPE == SAI_ROUTER_INTERFACE_TYPE_VLAN
+     * @condition SAI_ROUTER_INTERFACE_ATTR_TYPE == SAI_ROUTER_INTERFACE_TYPE_VLAN or SAI_ROUTER_INTERFACE_ATTR_TYPE == SAI_ROUTER_INTERFACE_TYPE_SUB_PORT
      */
     SAI_ROUTER_INTERFACE_ATTR_VLAN_ID,
```

Comment:
In the examples aren't the "vports" are all bridge ports? Are you indicating that a 1D rif can be used instead of a "vport" bridge port?